Makefile

```diff
@@ -231,6 +231,10 @@ rpc-check: rpc
 	for rpc in $$(find lnrpc/ -name "*.proto" | $(XARGS) awk '/    rpc /{print $$2}'); do if ! grep -q $$rpc lnrpc/rest-annotations.yaml; then echo "RPC $$rpc not added to lnrpc/rest-annotations.yaml"; exit 1; fi; done
 	if test -n "$$(git describe --dirty | grep dirty)"; then echo "Protos not properly formatted or not compiled with v3.4.0"; git status; git diff; exit 1; fi
 
+sample-conf-check: build
```

Comment:
ahha, this _is_ checking the dev binary. another option is to create a subtarget of `make install` that spits out prod binaries, then `make install` just copies them (more like how a C project would perform installation)

Reply:
Yeah, that wasn't optimal. I instead went with a `go run` approach that uses the same flags as the release build.